### PR TITLE
Adds division message endpoints.

### PR DESCRIPTION
**Pull request by randy-girard**

### CODE REVIEW

File: docs/collections/division_message_data.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Division Message Data
+
+## Methods
+
+- [loadDivisionMessageData](#loadDivisionMessageData)
+
+
```
Comment (amberwebb) [inline docs/collections/division_message_data.md]: Extra spaces?

File: src/collections/divisionMessageData.coffee [hunk 2]
```diff
@@ -0,0 +1,8 @@
+exports.loadDivisionMessageData = (params, callback) ->
+  if @isId params
+    params = teamId: params
+  else unless params and typeof params is 'object'
+    throw new TSArgsError 'teamsnap.loadDivisionMessageData', 'must provide a
+    teamId or query parameters'
+
+  @loadItems 'divisionMessageDatum', params, callback
```
Comment (amberwebb) [inline src/collections/divisionMessageData.coffee]: Should this be `divisionMessageData`? Isn't data plural of datum?

File: docs/collections/division_message_data.md (hunk 1 above)
Comment (randy-girard) [inline docs/collections/division_message_data.md]: Do you mean extra newline?

Comment (amberwebb) [inline docs/collections/division_message_data.md]: @randy-girard Never mind about that. Github has it formatted weird.

**Comment by amberwebb** [comment]
👍

**Comment by randy-girard** [comment]
@amberwebb I removed the division mark as read for now since I didn't add it at this point to v3. We can use the normal mark as read ,and it behaves the same way. https://github.com/teamsnap/teamsnap-javascript-sdk/pull/127/commits/4ef53af7874fbb5dcef4c7b2bb47d17f085c8ae9

**Comment by randy-girard** [comment]
I'm going to close this for now since I dont think its needed anymore.

**Pull request closed without merging**